src/core_plugins/console/public/src/controllers/sense_controller.js

```diff
@@ -11,23 +16,24 @@ module.run(function (Private, $rootScope) {
   };
 });
 
-module.controller('SenseController', function SenseController($scope, docTitle) {
+module.controller('SenseController', function SenseController($scope, $timeout, docTitle) {
 
   docTitle.change('Console');
 
-  // require the root app code, which expects to execute once the dom is loaded up
-  require('../app');
+  let input, output;
+  $timeout(() => {
+    output = getOutput($('#output'));
```

Comment:
I think it's worth making a comment here to reflect your note in the PR description:

```javascript
// We need to wait for the elements to be rendered before using jQuery to select them.
```